logs.h

```diff
@@ -84,6 +84,8 @@ class CLogs: public IGameEventListener2
 
 	unsigned int m_uiLastHealOnHit[34];
 
+	int m_iLastDirectHitVictim[34];
```

Comment:
Would change to unsigned int to keep consistency with similar variables and call it `m_uiLastDirectHitVictim`.